packages/slice-machine/lib/models/common/widgets/Group/type.ts

```diff
@@ -0,0 +1,20 @@
+import { Field, FieldType } from '../../CustomType/fields'
+import { SliceZone } from '../../CustomType/sliceZone';
+
+export type AsArray = ReadonlyArray<{key: string, value: Field}>
+export type AsObject = { [key: string]: Field | SliceZone }
+
+interface GroupConfig<T extends AsObject | AsArray> {
+  label: string,
+  placeholder: string,
+  fields: T
+}
+const defaultConfig = ({ label: '', placeholder: '', fields: [] as any })
```

Comment:
As discussed, there is something wrong about this. It works because we know that we never create a GroupField<AsObject>